example/src/screens/BasicFullscreenHandling.tsx

```diff
@@ -103,6 +103,7 @@ export default function BasicFullscreenHandling({
     <View style={styles.container}>
       <PlayerView
         player={player}
+        isFullscreen={false}
```

Comment:
nit: Do we need it here in the sample screen? With a hard-coded value I'm not sure it adds value here.

We could explain it the full screen feature guide.